lint/util.go

```diff
@@ -24,3 +25,14 @@ func nodeString(fset *token.FileSet, x ast.Node) string {
 	}
 	return buf.String()
 }
+
+func paramNamesStr(idents []*ast.Ident) string {
```

Comment:
Can you make this a method of `ParamDuplicationChecker`?

Reply:
I thought it could be useful for other chckers. It looks quite general :)

Reply:
We'll move it to `util.go` as soon as there is a second function user. :)
Let's be conservative about what makes it's way to `util.go`.

Reply:
Ok

Reply:
Done